examples/node_pool/variables.tf

```diff
@@ -60,6 +60,11 @@ variable "cluster_autoscaling" {
     max_cpu_cores       = number
     min_memory_gb       = number
     max_memory_gb       = number
+    gpu_resources = list(object({
```

Comment:
We will need to update some these defaults in examples

https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a9be73c07a740e597525bf5cf69b17a6752dd756/examples/node_pool/variables.tf#L64